pkg/common/consumer/consumer_handler.go

```diff
@@ -83,13 +85,15 @@ func (consumer *SaramaConsumerHandler) ConsumeClaim(session sarama.ConsumerGroup
 		if err != nil {
 			consumer.logger.Infow("Failure while handling a message", zap.String("topic", message.Topic), zap.Int32("partition", message.Partition), zap.Int64("offset", message.Offset), zap.Error(err))
 			consumer.errors <- err
+			consumer.handler.SetReady(false)
```

Comment:
If we go with the proposed definition for when a subscriber is ready, this is not needed.